src/components/shared/courses.js

```diff
@@ -8,10 +8,10 @@ function Courses ({ courseData, courseNotes, headingLevel }) {
 	var courseList="";
 	var trackCourseLevel="";
 	var coursesDisplay="<section class='row row-with-vspace'>";
-	var columnSetting = ((courseNotes === null || courseNotes === "") || (courseData === null || courseData === undefined)) ? "col-lg-12" : "col-lg-6";
-	
+	var columnSetting = ((courseNotes === null || courseNotes === "") || (courseData === null || courseData === undefined || courseData.length === 0)) ? "col-lg-12" : "col-lg-6";
```

Comment:
This is weird, because my suggestion was supposed to replace the line above once you committed it. Instead, it appears underneath, meaning `columnSetting` is defined twice

Reply:
Very odd! Fixed